code/modules/shuttle/supply.dm

```diff
@@ -330,11 +330,12 @@
 	slip.points = object.cost
 	slip.ordernumber = ordernum
 
-	var/stationName = (errors & MANIFEST_ERROR_NAME) ? new_station_name() : station_name()
+	var/error_name = "[pick("NCS","NSS","NDS","NAS","NBS","NXS")] [pick(greek_letters, phonetic_alphabet, numbers_as_words)]"
```

Comment:
You trashed the previous proc in favor of a tiny pick()?

Reply:
Why would we keep using the old proc? It basically did nothing useful, the names it made didn't even look like any of the station names in our lore. Honestly, with this the error is harder to notice, and is nicer.

Reply:
It was still better the other way.